src/shared/sol-conffile.c

```diff
@@ -285,9 +288,18 @@ _get_json_include_paths(
             continue;
         }
         if (sol_json_token_str_eq(&key, include_fallback, strlen(include_fallback))) {
-            *include_fallbacks = strndup(value.start + 1, sol_json_token_get_size(&value) - 2);
-            if (!*include_fallbacks)
+            char *buff;
+            struct sol_str_slice *s;
+            buff = strndup(value.start + 1, sol_json_token_get_size(&value) - 2);
```

Comment:
is this to remove quotes? There is a function to do that and should also handle escaping. Ask @bdilly